src/api/buddies.ts

```diff
@@ -41,10 +50,16 @@ const mappedStatuses: MappedStatuses = {
   Delete: 'deleted',
 };
 
+const mappedFromApi: MappedStatusesFromApi = {
+  banned: 'Banned',
+  deleted: 'Deleted',
+  ok: 'NotBanned',
+};
+
 const toBuddy = ({ id, display_name, status }: ApiBuddy): Buddy => ({
   buddyId: id,
   name: display_name,
-  status: status === 'banned' ? 'Banned' : 'NotBanned',
+  status: status ? mappedFromApi[status] : 'NotBanned',
```

Comment:
not sure but would assigning default value on line 59 to `status = 'ok'` and then just
`status: fromApiStatus[status]` or 
`status: fromApiStatus[status ?? 'ok']`

be simpler for brains. Not sure at all.

Reply:
Yeah, better for my brain.